testApp/scripts/ios/run-ui-tests-locally-ios.sh

```diff
@@ -0,0 +1,8 @@
+./scripts/ios/prepare-ui-tests-locally-ios.sh || exit 1
+
+cd ios && TEST=1 && RCT_NO_LAUNCH_PACKAGER=1 xcodebuild \
+-workspace DidomiExample.xcworkspace \
+-scheme DidomiExample \
+-sdk iphonesimulator \
+-destination "platform=iOS Simulator,name=iPhone 11,OS=14.5" \
```

Comment:
do you think adding a parameter for targeting a simulator or a real device is relevant?

Reply:
this device/simulator is used for running the tests so I think it's relevant because test results could be different depending on the screen size. About the OS version I think you are right and actually it might cause the test to break if the OS version that I use locally doesn't match what's on the remote Mac. I'll try removing the OS parameter and see how that behaves on the CI.